protos/sources/data-node/grpc-rest-bindings.yml

```diff
@@ -127,6 +127,12 @@ http:
       get: '/api/v2/upgrade/status'
     - selector: datanode.api.v2.TradingDataService.ListProtocolUpgradeProposals
       get: '/api/v2/upgrade/proposals'
+    - selector: datanode.api.v2.TradingDataService.GetOracleSpec
+      get: '/api/v2/oracle/spec'
```

Comment:
Looking at other patterns, this should take an id.

Reply:
Thanks, I fixed it.